modules/vault-cluster/outputs.tf

```diff
@@ -1,5 +1,7 @@
 output "asg_name" {
-  value = "${aws_autoscaling_group.autoscaling_group.name}"
+  # This is safe because asg_launch_mechanism will only allow one of aws_autoscaling_group.autoscaling_group.*
+  # or aws_autoscaling_group.lt_autoscaling_group.* to be non-empty.
+  value = "${join("",concat(aws_autoscaling_group.autoscaling_group.*.name,aws_autoscaling_group.lt_autoscaling_group.*.name))}"
```

Comment:
Our typical approach for this is: `${element(concat(x, y), 0)}`.

Reply:
Ah. Nice. Fixed.